src/components/grid-aware/DonationBlock/DonationBlock.module.css

```diff
@@ -74,6 +74,18 @@
   margin-top: 5px;
 }
 
+.mainTitleImage {
+  height: auto;
+  margin-bottom: 0;
```

Comment:
Could you increase the bottom margin? Compared to the normal `.mainTitle`, this ends up having less effective space between it and the subtitle. The `.mainTitle` has a font size of 50px and a line height of 1.3 = 65px, which means that there's actually (65-50)/2 = 7.5px of effective space between the baseline of the text and the bottom of the element.

Without additional spacing, it makes the page feel cramped.

Reply:
I used 10px margin-bottom